compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```
@@ -708,10 +709,15 @@ object messages {
 
     private val actualArgString = actual.map(_.show).mkString("[", ", ", "]")
 
-    private val prettyName = fntpe.termSymbol match {
-      case NoSymbol => fntpe.show
-      case symbol   => symbol.showFullName
-    }
+    private val prettyName =
+      try
+        fntpe.termSymbol match {
+          case NoSymbol => fntpe.show
+          case symbol   => symbol.showFullName
+        }
+      catch {
+        case NonFatal(ex) => fntpe.show
```

Sure, but we shouldn't be throwing any kind of exceptions, and silently catching them all might hide issues.